### PR TITLE
Add a boilerplate blueprint to README

### PR DESCRIPTION
**Pull request by nick-stroud**
This is meant to help when writing a blueprint from scratch.

Proposed merging to `release-candidate` since it is documentation only.

### Submission Checklist

* [x] Have you installed and run this change against pre-commit? (`pre-commit
  install`)
* [x] Are all tests passing? (`make tests`)
* [x] Have you written unit tests to cover this change?
* [x] Is unit test coverage still above 80%?
* [x] Have you updated all applicable documentation?
* [x] Have you followed the guidelines in our Contributing document?

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -629,6 +630,28 @@ The blueprint file is composed of 3 primary parts, top-level parameters,
 deployment variables and deployment groups. These are described in more detail
 below.
 
+### Blueprint Boilerplate
+
+The following is a template that can be used to start writing a blueprint from
+scratch.
+
+```yaml
```
Comment (heyealex) [inline examples/README.md:638]: I could see a good argument for creating a separate file for this in examples, that makes it one step easier to start creating a blueprint from scratch by either copying the file or just working in the template directly.

That could also be added at a later time, so I won't hold up the PR on it.

**Review by heyealex — APPROVED** [review]
I added a couple comments with suggestions, but none of them are mandatory so feel free to merge if you prefer it the way it is.

File: examples/README.md (hunk 1 above)
Comment (nick-stroud) [reply examples/README.md:638]: Ack.

File: examples/README.md [hunk 2]
```diff
@@ -629,6 +630,28 @@ The blueprint file is composed of 3 primary parts, top-level parameters,
...
+---
+blueprint_name: ...
+
+vars:
+  project_id: ...
```
[the remark this answers is not in the document]
Comment (nick-stroud) [reply examples/README.md]: Updated.

**Review by heyealex — APPROVED** [review]

**Pull request merged**